spec/controllers/scim_rails/scim_groups_controller_spec.rb

```diff
@@ -243,6 +283,66 @@
     end
   end
 
+  describe "delete" do
+    let(:company) { create(:company) }
+
+    context "when unauthorized" do
+      before { delete :delete, { id: 1 } }
+
+      it "returns scim+json content type" do
+        expect(response.content_type).to eq "application/scim+json"
+      end
+
+      it "fails with no credentials" do
+        expect(response.status).to eq 401
+      end
+
+      it "fails with invalid credentials" do
+        request.env['HTTP_AUTHORIZATION'] = ActionController::HttpAuthentication::Basic.encode_credentials("unauthorized","123456")
```

Comment:
Where is this usually set?
This should ideally be set with a `let`, but not sure what we have access to here

Reply:
`request.env` is a special thing in rspec. And by "special", I mean in the bad way. If it works, it's best to leave it be 😬